### DISCUSSION TITLE
Bug maybe? Inconsistency secondary button in Delete pattern (Paste website)

### DISCUSSION THREAD

**Discussion by frankfzhao** [twilio-labs/paste#2785] [General] [answered]
Hey DSYS team, I found this inconsistency in the Paste website. 
<img width="992" alt="Screenshot 2022-10-31 at 8 40 15 AM" src="https://user-images.githubusercontent.com/82524550/199034976-61bd9ad2-127a-4c2b-9158-a86bb114a816.png">
In the delete pattern page, the "Cancel" button is in ghost button style using the primary CTA blue color while the default secondary button style is in gray color. Is this a bug or just the style we want to use for delete? thanks

**Comment by loreina** [comment]
Hi @frankfzhao,

Thanks for catching this! It looks like the screenshots on that page are outdated and using an old secondary button. I created a ticket in our backlog to update that page with live examples and the current design of our secondary button.

Here's a screenshot of what it should look like:

<img alt="image" src="https://user-images.githubusercontent.com/23108291/199085362-6c8944c5-0ae6-4d4a-bfd4-d440709f73fe.png">

The team will follow up once the docs site is updated. Thanks!

**Comment by SiTaggart** [comment, marked as answer]
Delete pattern examples were all replaced with coded examples so this should be resolved now.
